captalize_decorate.rb

```diff
@@ -0,0 +1,7 @@
+require './decorate'
```

Comment:
- [ ] Kindly ensure you test your code. This import is incorrect. There is not file named `decorate`. What you have is `decorator`. Kindly fix that.